aries-site/src/examples/components/spinner/ListSpinnerExample.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Box, List, Spinner, Text } from 'grommet';
+import {
+  Console,
+  PersonalComputer,
+  Desktop,
+  User,
+  System,
+} from 'grommet-icons';
+
+const data = [
+  {
+    name: 'Eric Soderberg',
+    option: 'Active',
+    icon: <PersonalComputer />,
+  },
+  {
+    name: 'Shimrit Yacobi',
+    option: 'Inactive',
+    icon: <User />,
+  },
+  {
+    name: 'Chris Carlozzi',
+    option: 'Active',
+    icon: <Desktop />,
+  },
+  {
+    name: 'Matthew Glissmann',
+    option: 'Active',
+    icon: <System />,
+  },
+  {
+    name: 'Taylor Seamans',
+    option: 'Inactive',
+    icon: <Console />,
+  },
+];
+
+export const ListSpinnerExample = () => (
+  <Box gap="small" width={{ max: 'xxlarge' }} margin="auto" fill>
+    <List
+      background="background-front"
+      data={data}
+      action={item => <Text>{item.option}</Text>}
+      onClickItem={e => {
+        // eslint-disable-next-line no-alert
+        alert(`You clicked on ${e.item.name}`);
+      }}
+    >
+      {(datum, index) => (
+        <Box direction="row" gap="small" align="center" key={index}>
+          {datum.icon}
+          <Text weight="bold">{datum.name}</Text>
+        </Box>
+      )}
+    </List>
+    <Box gap="small" alignSelf="center" direction="row-responsive">
+      <Spinner
+        border={[
+          { side: 'all', color: 'background-contrast', size: 'medium' },
+          { side: 'right', color: 'brand', size: 'medium' },
```

Comment:
I think the color the designs call for is `green` instead of `brand`

Reply:
I would hope that the Spinner is styled correctly via the theme. Can't we just use `<Spinner />` here?

Reply:
I was trying to but `border` in the theme under `spinner container` but it wasn't working correctly going to look more at it today

Reply:
> I would hope that the Spinner is styled correctly via the theme. Can't we just use `<Spinner />` here?

moved to theme!